vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go

```diff
@@ -166,6 +169,17 @@ const (
 	ContainerScalingModeOff ContainerScalingMode = "Off"
 )
 
+// ContainerLimitScalingMode controls whether autoscaler limit scaling
+// is enabled for a specific container.
+type ContainerLimitScalingMode string
+
+const (
+	// ContainerLimitScalingModeProportional means limits are scaled automatically.
+	ContainerLimitScalingModeProportional ContainerLimitScalingMode = "Auto"
```

Comment:
I'd like the const name and value to be consistent. So to be consistent with ContainerScalingMode:

// ContainerLimitScalingModeAuto means limits are scaled automatically.
// Limit is scaled proportionally to the request.
ContainerLimitScalingModeAuto ContainerLimitScalingMode = "Auto"